gulpfile.js

```diff
@@ -259,6 +263,15 @@ function cssWatch(cb) {
     cb();
 }
 
+/**
+ * Transpile ES6 JavaScript files.
+ */
+function transpileJs() {
+    return gulp.src('media/js/components/*.es6.js')
```

Comment:
We probably want to avoid copying these `*.es6.js` files over to `static_final` without being compiled first: https://github.com/mozilla/bedrock/blob/master/gulpfile.js#L375